src/renderer/commands/drawObject.ts

```diff
@@ -33,11 +34,21 @@ export interface DrawObjectProps {
 }
 
 /*
- * Shader to draw a single object with Phong shading to the screen
+ * Shader to draw a single object with Phong shading to the screen.
+ *
+ * @param {REGL.regl} regl: The regl object we're drawing.
+ * @param {number} numLights: The number of light points we want filled.
+ * @param {number} maxLights: The maximum number of light points that may exist.
  */
 export function drawObject(
-    regl: REGL.regl
+    regl: REGL.regl,
+    numLights: number = 1,
+    maxLights: number = 1
 ): REGL.DrawCommand<REGL.DefaultContext, DrawObjectProps> {
+    if (numLights > maxLights) {
+        throw new RangeError('numLights must be less than or equal to maxLights.');
```

Comment:
Can we put the actual number of lights at the end of the error message? And maybe the requested number of lights?
I'm not sure if it's bad practice to make the error message dynamic like that, but I think it would be useful to have
"numLights (24) must be less than or equal to maxLights (20)"